### ISSUE TITLE
Volume Control

### ISSUE THREAD

**Issue by ponychicken** [pdherbemont/Glasses#39]
Implement a lunettes-intern volume control and normalisation.

**Comment by pdherbemont** [comment]
Ought to be easy, however, I am not certain that libvlc and libvlc core support a per mediaPlayer volume control. But that's something we could fix later.

**Comment by fkuehne** [comment]
I don't think this is possible with libvlc right now, although the core likely supports that...

**Comment by pdherbemont** [comment]
This should be doable now.

**Comment by pdherbemont** [comment]
Marking as blocker as we have and will have too many complaint and this is a low hanging fruit. We could downgrade it though.
